doc/release/platforms/README.cray

```diff
@@ -157,14 +178,38 @@ Using Chapel on a Cray System
           source util/setchplenv.bash
 
 
-2) Compile your Chapel program.  For example:
+2) Optionally, set one or more of the following environment variables to
+   select a Chapel configuration.  These are described in greater detail
+   in README.chplenv.
+
+     CHPL_LOCALE_MODEL : desired locale model, default "flat"
+     CHPL_TASKS        : tasking implementation, default "muxed" on Cray
+                         XC/XE with pre-built module, else "qthreads"
```

Comment:
the defaults seem more appropriate here. chpl_tasks should probably say that fifo is used for target compiler cray